packages/components-native/src/Glimmer/Glimmer.style.ts

```diff
@@ -5,7 +5,7 @@ export const shineWidth = tokens["space-largest"];
 
 export const styles = StyleSheet.create({
   container: {
-    backgroundColor: tokens["color-surface--background"],
+    backgroundColor: "rgba(0, 0, 0, 0.05)",
```

Comment:
@MichaelParadis this is the same issue we have in web, where we'll probably need to bake in dark/light values within the component